### ISSUE TITLE
Website title would also be nice.  I use IPFS for many sites, and seeing the origin alone doesn't really help me a whole lot.

### ISSUE THREAD

**Issue by KillariDev** [DarkFlorist/TheInterceptor#183]
Website title would also be nice.  I use IPFS for many sites, and seeing the origin alone doesn't really help me a whole lot.

_Originally posted by @MicahZoltu in https://github.com/DarkFlorist/TheInterceptor/pull/178#pullrequestreview-1304499692_

**Comment by KillariDev** [comment]
Need to think about how to show this from security perspective. Icon can also already be bit dangerous as people look at it, and not the URL.

**Comment by MicahZoltu** [comment]
Agreed.  I definitely think we should include the origin prominently.  That being said, presumably the user doesn't have a scam version of Uniswap open in one tab and a non-scam version open in another tab... right?  😬 

We could do some work to maintain a list of IPFS hashes for valid dapps, another bit of useful curation work we could provide.  Then we could give a friendly name to well known IPFS pages.

**Comment by KillariDev** [comment]
![image](https://user-images.githubusercontent.com/13102010/220265789-cf3c23dd-b46a-4cb8-aced-9baad468293b.png)

I have this UI design on access requests, cannot fit all that info on every page thought

**Comment by KillariDev** [comment]
this is implemented everywhere. Both Url and title is shown
